api-reference/openapi.json

```diff
@@ -869,6 +869,130 @@
         }
       }
     },
+    "/v1/projects/{id}/counters": {
+      "get": {
```

Comment:
should we include "operationId" for code gen?

Reply:
yeah i didnt do it on that part but ill add it